google/earthenterprise
# Add: Consistent Versioning of RPMs

Use the long version as the basis of the RPM filenames.

Test Procedure:
- Build and Stage-Install.
- Note the new file: /tmp/fusion_os_install/gee_long_version.txt
- Cd to earth_enterprise/rpms/
- Run: ./gradlew -PbuildOpenGee
- ls build/distributions/*rpm

Check and compare that filenames are as expected. You may edit /tmp/fusion_os_install/gee_long_version.txt and re-build with Gradle to check additional values.

Example expectations (the el6 part may change based on your OS):
5.2.1-2.release --> opengee-server-5.2.1.1.el6.rpm
5.2.2-0.alpha.DATESTAMP.GITHASH --> opengee-server-5.2.2.0.alpha.DATESTAMP.GITHASH-1.el6.rpm
5.2.2-1.beta --> opengee-server-5.2.2.1.beta-1.el6.rpm
5.2.2-2.beta.DATESTAMP.GITHASH --> opengee-server-5.2.2.2.beta.20171208.hashhash-1.el6.rpm
5.2.2-4.release --> opengee-server-5.2.2-1.el6.rpm
